Makefile

```diff
@@ -43,7 +43,6 @@ install: ## Install all binaries
 .PHONY: lint
 lint: $(BIN)/golangci-lint $(BIN)/buf ## Lint Go and protobuf
 	test -z "$$($(BIN)/buf format -d . | tee /dev/stderr)"
-	$(GO) vet ./...
```

Comment:
https://github.com/golang/protobuf/issues/1658

Reply:
think we can use `golangci-lint` to run `go vet` anyway, so good to consolidate.